notebooks/240-dolly-2-instruction-following/240-dolly-2-instruction-following.ipynb

```diff
@@ -18,6 +18,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        ov_model = OVModelForCausalLM.from_pretrained(model_id, device=current_device, from_transformers=True)
</pre><p>please add converting model to half()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Added